scripts/installation/bin/yb-ctl

```diff
@@ -1413,7 +1425,10 @@ class ClusterControl:
         logging.info("Stopping {} PID={}".format(name, pid))
         # Kill the process, if it exists.
         try:
-            os.kill(pid, signal.SIGTERM)
+            if self.options.force:
+                os.kill(pid, signal.SIGKILL)
```

Comment:
Do `SIGQUIT` instead if it's the postmaster process (e.g. `name.startswith("Postmaster")`).  I gave my reasons in comments on the issue and PR.